src/app/component/quiz/quiz.component.ts

```diff
@@ -122,11 +122,14 @@ export class Quizcomponent implements OnInit, OnDestroy {
 
   next(questionindex: number): void {
     this.carousel.next();
+    if (this.dialogService.hasModelOpen()) {
+      this.dialogService.destroy();
+    }
     const values = this.FormArray.controls[questionindex].value.radioValue;
     this.answer(questionindex, values);
```

Comment:
Please use camelcase here and also add undefined check here at line no. 128

Reply:
Yes i have used camelcase and added undefined check